documentation/user_docs/docs/manual/Symmetrising_etc.rst

```diff
@@ -124,76 +279,104 @@ Rebin the sqw object ``win`` with the boundaries (and projection axes) of the te
 
 
 Symmetrise data, then unfold back to original range
-***************************************************
+===================================================
 
-**N.B.** For producing plots only, any analysis on these results will be invalid due to double / treble / etc. counting of data.
+**N.B.** For producing plots only, any analysis on these results will be invalid due to double / treble / etc. counting
+ of data.
 
-Below we show a script that uses the ``symmetrise_sqw`` and ``combine_sqw`` commands to fold a dataset and then unfold. In the example we have a constant energy slice in the (h,k)-plane which we fold twice to obtain the positive quadrant. We then unfold the result in the opposite direction and combine with the positive quadrant, then unfold this and combine. This produces an image which covers all four quadrants with suitably folded versions of just one of the quadrants.
+Below we show a script that uses the ``symmetrise_sqw`` and ``combine_sqw`` commands to fold a dataset and then
+unfold. In the example we have a constant energy slice in the (h,k)-plane which we fold twice to obtain the positive
+quadrant. We then unfold the result in the opposite direction and combine with the positive quadrant, then unfold this
+and combine. This produces an image which covers all four quadrants with suitably folded versions of just one of the
+quadrants.
```

Comment:
I have not grasped what the result here is. What is duplicated, and what is just moved?

Also, the use of fold here is confusing with respect to a rotational nfold. Do you have an alternative description?